packages/web/src/components/Layout/UserProfileLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Heading, HStack, Spacer, useTheme, VStack, Text, Box } from '@chakra-ui/react';
+
+export interface UserProfileData {
+  name: string;
+  pronouns?: string;
+  location?: string;
+  hireable: boolean;
+  purpose: string;
+  schoolName?: string;
+  major?: string;
+  graduationDate?: Date;
+}
+
+export const UserProfileLayout: React.FC<UserProfileData> = ( user: UserProfileData ) => {
```

Comment:
I apologize. I could not finish a UI for the profile page in time.

Current profile page just look like:
Name (big font)
pronouns (smaller font)
school name (smaller font)